app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -12,6 +12,47 @@ $mobile-breakpoint: 700px;
   --space-6: calc(0.25rem * 6);
 }
 
+.admin-site-settings {
+  &__login,
```

Comment:
Instead of doing all these individual ones, could you create a single `__not-migrated` class and apply that with `getAdminBodyClass`?

Reply:
Or do you want to keep these more specific?